manageiq-providers-ansible_tower.gemspec

```diff
@@ -14,6 +14,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,lib}/**/*"]
 
   s.add_runtime_dependency "ansible_tower_client", "~> 0.19"
+  # s.add_runtime_dependency "ansible_tower_client", "~> 0.20"
```

Comment:
This PR will fail until the new client is released containing the changes in https://github.com/ansible/ansible_tower_client_ruby/pull/124